index.js

```diff
@@ -54,7 +68,16 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+let sum = 0
+let average = 0
```

Comment:
Why create this variable here only to reassign it after you've determined `sum`? You could have just returned `sum / movies.length` or declared `average` and assigned it to `sum / movies.length` at the same time.